src/util/getPageCategory.ts

```diff
@@ -16,7 +16,7 @@ const categories = [
  * @param url URL for the current page.
  * @returns The category for the current page as used by Algolia DocSearch to group search results.
  */
-export function getPageCategory(url: URL) {
+export function getPageCategory(url: { pathname: string }) {
 	const langAgnosticPath = url.pathname.replace(/\/\w\w(-\w\w)?\//, '');
 	for (const [path, label] of categories) {
 		if (langAgnosticPath.startsWith(path)) return label;
```

Comment:
I'm not sure if this is a dev-only behavior, but the switch to the Reference tab isn't working for the Errors pages. The "langAgnosticPath" is starting with `C:/` for me, that's why it is unable to match with `path`. 

Considering we add the final `/` to the categories I guess our chances of ending up causing a route conflict are small, so a fast fix for this would be to change the string method to `includes` instead:

```suggestion
		if (langAgnosticPath.includes(path)) return label;
```

Reply:
Oh I think this is because of one place that uses `import.meta.url`!

Reply:
Pushed a small change that I think should fix this. (Not 100% sure because I think it was a Windows-only thing maybe.)